### DISCUSSION TITLE
dev and distribution docs

### DISCUSSION THREAD

**Discussion by j042** [tataratat/splinepy#304] [General]
Is there a way to host two version of API docs? @clemens-fricke

**Comment by clemens-fricke** [comment]
What would be the difference between the versions?

**Reply by j042** [reply]
dist would be the version that you get from `pip install splinepy` and dev would be the current main branch

**Reply by clemens-fricke** [reply]
On read the docs? 

Should be possible. See the docs for godot engine they have a version selector with latest/stable. If you wish I could look into how that works and what selectors you can have there.

**Reply by j042** [reply]
please! that would be great

**Comment by clemens-fricke** [comment]
In general, this should be possible, but I am not sure how to do it via GitHub pages like you are currently publishing the docs with. The precious answer was when I assumed you published via readthedocs. 

I have not worked with GitHub pages before.

**Reply by j042** [reply]
We'd build twice: first with current pypi version and second with current main version. Then we'd put files from the latter to the former and create a link to access them. All by leveraging your html and web development skills :D
